DEPS

```diff
@@ -1239,5 +1239,20 @@ hooks = [
       '--as-gclient-hook',
       Var('mac_sdk_min')
     ]
-  }
+  },
+  # Track the hash of this DEPS file.
+  # Used by the engine tool to determine whether to sync again.
+  # Keep this step last.
+  {
+    'name': 'Update DEPS hash file',
+    'pattern': '.',
+    'action': [
+      'python3',
+      'src/flutter/build/sha256.py',
+      '--source',
+      'src/flutter/DEPS',
+      '--output',
+      'src/flutter/build/DEPS.sha256'
```

Comment:
@zanderso Is this the best location for this hash file? I "hid" it in the `//flutter/build` directory so that the `//flutter/` directory isn't "cluttered" with this implementation detail.

Reply:
I'd slightly prefer a dot file directly under `//flutter` like `//flutter/.DEPS.sha256`.